zip-0226.rst

```diff
@@ -252,12 +262,10 @@ Senders must not be able to change the Asset Base for the output note in a Split
     - Replace the input note value by a generic value, :math:`\mathsf{v}'`, as :math:`\mathsf{cv^{net}} = \mathsf{ValueCommit_rcv^{OrchardZSA}(v’ - v^new, \mathsf{AssetBase}^{\mathsf{Orchard}}_{\mathsf{AssetId}})}`
 - Add a boolean :math:`\mathsf{split\_flag}` variable as an auxiliary witness. This variable is to be activated :math:`\mathsf{split\_flag} = 1` if the Action in question has a Split Input and :math:`\mathsf{split\_flag} = 0` if the Action is actually spending an input note:
     - If :math:`\mathsf{split\_flag} = 1` then constrain :math:`\mathsf{v}' = 0` otherwise constrain :math:`\mathsf{v}'=\mathsf{v^{old}}` from the auxiliary input.
-    - If :math:`\mathsf{split\_flag} = 1` then constrain :math:`\mathsf{v^{old}} \neq 0`.
+    - If :math:`\mathsf{split\_flag} = 1` then constrain :math:`\mathsf{is\_native\_asset}' = 0` because split notes are only available for Custom Assets.
```

Comment:
Just confirming that the $\mathsf{v^{old}} \neq 0$ constraint is no longer required?

Reply:
Yes this constraint is not longer required